### PR TITLE
Added ability to add labels to users without rights

### PR DESCRIPTION
**Pull request by tavriaforever**
@tormozz48

### CODE REVIEW

File: server/forum.js [hunk 1]
```diff
@@ -121,6 +123,11 @@ module.exports = function(pattern, options) {
         } else {
             var result = {};
 
+            if(query.__access === 'owner' && ownerToken) {
```
Comment (tormozz48) [inline server/forum.js:126]: Can we move this code to forum initialization? For example to line 21?

File: desktop.blocks/forum/forum.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ modules.define('forum', ['i-bem__dom', 'jquery'], function(provide, BEMDOM, $) {
         _subscribes: function() {
             this._formAdd = this.findBlockInside('add-form', 'forum-form');
             this._formAdd && this._formAdd.on('submit', this._addIssue, this);
+            this._forumUrl = this._formAdd.params['forumUrl'] || '/';
```
Comment (tadatuta) [inline desktop.blocks/forum/forum.js]: why not to use dot notation like `this._formAdd.params.forumUrl`?

File: server/forum.js [hunk 3]
```diff
@@ -17,6 +17,8 @@ module.exports = function(pattern, options) {
     template.init(options);
     model.init(options);
 
+    var ownerToken = options['owner_token'];
```
Comment (tadatuta) [inline server/forum.js]: `options.owner_token`

File: server/forum.js (hunk 1 above)
Comment (tavriaforever) [inline server/forum.js:126]: No, because we must check it every ajax request.

File: desktop.blocks/forum/forum.js (hunk 2 above)
Comment (tavriaforever) [inline desktop.blocks/forum/forum.js]: There is no special reason for it, so i can replace with dot notation

File: server/forum.js (hunk 3 above)
Comment (tavriaforever) [inline server/forum.js]: ok

**Pull request merged**